project.clj

```diff
@@ -9,19 +9,19 @@
 
   :dependencies [[aleph "0.4.6"]
                  [clj-time "0.15.1"]
-                 [danlentz/clj-uuid "0.1.7"
+                 [danlentz/clj-uuid "0.1.9"
```

Comment:
How is it relevant to this PR ?

Reply:
Ah, it's really not. There were just a couple things that looked out of date so bumped them.  Maybe better not to.  If it's better to leave uuid as-is, I'll remove this change.